srv/modules/runners/disk_led.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+"""
+This runner is used to turn on/off the ident and fault LED of a disk.
+"""
+from __future__ import absolute_import
+
+import logging
+import pprint
+import re
+
+import salt.client
+
+log = logging.getLogger(__name__)
+
+
+def _process(hostname, device_name, led, status):
+    """
+    Turn on/off the specified LED of the disk on the given host.
+    """
+    local_client = salt.client.LocalClient()
+    device_file = '/dev/{}'.format(device_name)
```

Comment:
As mentioned in https://github.com/SUSE/DeepSea/pull/1474/files#r260701496 , add format checks here.

Reply:
Done